### PR TITLE
[master] EMBL updates

### PR DESCRIPTION
**Pull request by IvarsKarpics**

### CODE REVIEW

File: HardwareObjects/EMBL/EMBLBeamFocusing.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ def init(self):
         """Reads available focusing modes from the config xml and
            attaches corresponding motors
         """
+
+        self.cmd_set_calibration_name = self.getCommandObject(\
+            'cmdSetCallibrationName')
```
Comment (rhfogh) [inline HardwareObjects/EMBL/EMBLBeamFocusing.py]: Are you sure you want 'cmdSetCallibrationName') with a double 'l'?

File: HardwareObjects/EMBL/EMBLBeamlineTest.py [hunk 2]
```diff
@@ -1055,11 +1067,11 @@ def center_beam_task(self):
 
                 if abs(delta_hor) > 0.001:
                     log.info("Beam centering: Moving horizontal by %.4f" % delta_hor)
-                    self.horizontal_motor_hwobj.moveRelative(delta_hor, timeout=5)
+                    self.horizontal_motor_hwobj.mov_relative(delta_hor, timeout=5)
```
Comment (rhfogh) [inline HardwareObjects/EMBL/EMBLBeamlineTest.py]: Are you sure of the spelling in 'mov_relative'? Not missing an 'e'?

Comment (IvarsKarpics) [reply HardwareObjects/EMBL/EMBLBeamlineTest.py]: Opaa.. thanks for seeing this...

File: HardwareObjects/EMBL/EMBLBeamFocusing.py (hunk 1 above)
Comment (IvarsKarpics) [reply HardwareObjects/EMBL/EMBLBeamFocusing.py]: Fixed... Thanks again for a detailed review!

File: HardwareObjects/EMBL/EMBLMotorsGroup.py [hunk 3]
```diff
@@ -226,33 +235,35 @@ def positions_changed(self, positions):
...
+        return self.detected_foc_mode
 
     def status_changed(self, status):
         """Called if motors status is changed. Pysignal with new
            status has been sent"""
-        values_to_send = {}
         for motor in self.motors_list:
             old_status = motor['status']
-            if type(status) == list or type(status) == tuple:
+            if type(status) in (tuple, list):
```
Comment (rhfogh) [inline HardwareObjects/EMBL/EMBLMotorsGroup.py]: How about:
if isinstance(status, (tuple, list)):
That allows subtypes of either (e.g. namedtuples). One could get pedantic and test for 
isinstance (x, collections.Sequence) and not isinstance (x, (str, byte))
but that would seem ot be overkill.

Comment (IvarsKarpics) [reply HardwareObjects/EMBL/EMBLMotorsGroup.py]: well, it is a mater of taste. I changed code to use isinstance.

**Pull request merged**